manifest

```diff
@@ -84,55 +83,75 @@ cd  "$__type/files"
 ################################################################################
 # SSL / HTTPs configuration
 #
-if [ -f "$__object/parameter/ssl" ]; then
-    ssl_base="$__type/files/ssl"
-
-    # Select the ssl certificate + key
-    if [ -f "$__object/parameter/ssl-name" ]; then
-        ssl_name="$(cat "$__object/parameter/ssl-name")"
-    else
-        echo "Please select ssl certificate" >&2
-        exit 1
-    fi
-
-if [ -f "$__object/parameter/ssl-cert" -a  -f "$__object/parameter/ssl-key" ]; then
-    else
-		if [ -f "$__object/parameter/ssl-name"]; then
-		else
-		echo "Missing parameter" >&2
-		exit 1
-fi
-
-    if [ -f "$__object/parameter/ssl-cert" ]; then
-        
-    else
-        ssl_cert="/etc/ssl/certs/${ssl_cert}.crt"
-    fi
+if [ -f "$__object/parameter/ssl" ] || [ ! -f "$__object/parameter/dh" ]; then
```

Comment:
Doesn't work
You should only check if ssl is required
then check all the needed parameters